app/gen-server/lib/DocWorkerMap.ts

```diff
@@ -306,6 +310,38 @@ export class DocWorkerMap implements IDocWorkerMap {
     }
   }
 
+  public onWorkerUnavailable(workerInfo: DocWorkerInfo, cb: () => void): () => void {
```

Comment:
I think it can be easily unit-tested using this tip:
1. Instantiate some stub class which contains a client property (also a stub);
2. call : 
```ts
DocWorkerMap.prototype.onWorkerUnavailable.call(stub,
    {id: 'WORKER-ID', group: 'GROUP', internalUrl: 'whatever', publicUrl: 'whatever'},
    spy
);
```

I'll try to implement this test.